### PR TITLE
Week 3

### PR DESCRIPTION
**Pull request by Rajeev2k11**
Week 3 assignmnet

### CODE REVIEW

File: Week-3/JS Scopes, Global Object & Context/Assignment-1.js
Comment (NishantNaithani) [inline Week-3/JS Scopes, Global Object & Context/Assignment-1.js:1]: Here,
Use this method to calculate the power `return Math.pow(num1, exp);`

File: Week-3/Javascript Fundamentals/assignment-1.js
Comment (NishantNaithani) [inline Week-3/Javascript Fundamentals/assignment-1.js:1]: Here,
`.map` is used for iterating and creating a purely new array with modifications, you should use `.forEach` for these types of operations where you just need to iterate the array and update other variables

**Review by NishantNaithani — CHANGES_REQUESTED** [review]

File: Week-3/JS Scopes, Global Object & Context/Assignment-1.js
Comment (Rajeev2k11) [reply Week-3/JS Scopes, Global Object & Context/Assignment-1.js:1]: done

File: Week-3/Javascript Fundamentals/assignment-1.js
Comment (Rajeev2k11) [reply Week-3/Javascript Fundamentals/assignment-1.js:1]: done

**Pull request merged**